how-to/security/apparmor.md

```diff
@@ -225,6 +211,43 @@ Profiles are meant to provide security and so can't be too permissive. But often
   * To mitigate the drawbacks of above approaches, **local includes** were introduced, adding the ability to write arbitrary rules that not run into issues during upgrades that modify the packaged rule.
   * The files can be found in `/etc/apparmor.d/local/` and exist for the packages that are known to sometimes need slight tweaks for special setups.
 
+
+## Disabling or Re-enabling AppArmor
+
+Starting with Ubuntu 24.04 and later, the AppArmor services are baked into the Ubuntu Kernel, and require special steps to fully disable or re-enable.
+
+### Disable AppArmor
+
+> WARNING! Disabling AppArmor reduces the security of your system! You should only disable apparmor if you understand the security implications 
+> of disabling the service!
+
+To disable AppArmor, you must do the following:
+
+* Stop and disable the `apparmor.service` in SystemD to prevent AppArmor profiles from being loaded:
+
+    ```
+    sudo systemctl stop apparmor.service
+    sudo systemctl disable apparmor.service
+    ```
+
+* Edit `/etc/default/grub` and add `apparmor=0` to `GRUB_CMDLINE_LINUX` in `/etc/default/grub`
+
+* Run `sudo update-grub` to refresh the boot configuration
+* Reboot the system.
+
+### Re-enable AppArmor
```

Comment:
maybe we should mention how an enabled apparmor looks like
How about adding as example:

```
$ sudo aa-status
apparmor module is loaded.
119 profiles are loaded.
24 profiles are in enforce mode.
   /usr/bin/man
...
```

Reply:
see https://github.com/canonical/ubuntu-server-documentation/pull/113#discussion_r1915515239